src/foam/box/sf/SFManager.js

```diff
@@ -0,0 +1,276 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.box.sf',
```

Comment:
Is it possible to have the standard SFManager, then a medusa version in the medusa package?  
And we'll control via nspec.  So medusa could  replace the standard 'sfManager' nspec for example.

Reply:
Yes, I can do that